frontend/src/app/shopping-cart/shopping-cart.component.ts

```diff
@@ -33,4 +34,16 @@ export class ShoppingCartComponent implements OnInit {
     this.shoppingCart.updateItem(item);
   }
 
+  private totalPrice(): number {
+    return 0; // todo
+  }
+
+  private checkout() {
+    return; // todo
+  }
+
+  private clearCart() {
+    this.items.forEach(item => this.deleteItem(item));
```

Comment:
This should be done in the shopping cart service. There is currently not implemented a method for it, but that should be easy enough to do.